model-engine/customer_service_data.csv

```diff
@@ -0,0 +1,2 @@
+prompt,response

```

Comment:
is this used in a test case or documentation? if not, remove

Reply:
oh yeah this shouldn't have been committed